frameworks/Elixir/phoenix/web/controllers/page_controller.ex

```diff
@@ -31,7 +34,8 @@ defmodule Hello.PageController do
     end
 
     conn
-    |> json(Enum.map(1..q, fn _ -> Repo.get(World, :rand.uniform(10000)) end))
+    |> put_resp_content_type("application/json", nil)
+    |> send_resp(200, Poison.encode!(Enum.map(1..q, fn _ -> Repo.get(World, :rand.uniform(10000)) end)))
```

Comment:
Are we required per the spec to perform multiple queries? Or can we perform a single one?